fapolicy_analyzer/ui/changeset_wrapper.py

```diff
@@ -42,13 +42,38 @@ def deserialize(cls, data: Union[Dict[str, str], str]) -> "Changeset":
                     tcs.delete(path)
             return tcs
         elif isinstance(data, str):
-            rcs = RuleChangeset()
-            rcs.parse(data)
-            return rcs
+            try:
+                rcs = RuleChangeset()
+                rcs.parse(data)
+                return rcs
 
+                try:
```

Comment:
Whats going on here?

Reply:
It's trying to load the changeset data and now we have 2 `Changeset` that return as string and the `RuleChangeset` has additional exceptions in the rust layer (I think its something about having 0 rules)

Reply:
It was but not like that. Good catch.